sencha-workspace/packages/slate-cbl/src/view/tasks/TaskForm.js

```diff
@@ -253,6 +253,9 @@ Ext.define('Slate.cbl.view.tasks.TaskForm', function() {
                         : Ext.String.format(me.getInitialConfig('editTitle'), task.getId(), task.get('Title'))
                 );
 
+                // Hide the parentTask field if this task is a parent of other tasks
+                me.getParentTaskField().setHidden(task.get('ChildTasks').length > 0);
```

Comment:
@BillClinton it looks like the reason this isn't working is because ChildTasks is dynamically populated in the main view while a fresh task instance is loaded from the server when opening the modal in order to grab some additional embedded data and that load doesn't include the dynamic field `SubTasks` or the populated `ChildTasks` that the main store has.

The reason `ChildTasks` is populated client side rather than using the server-side `SubTasks` is because arbitrary filters can be applied to the set of tasks loaded into the main view, and we want to only show the child tasks that are part of that filtered set rather than all subtasks of any task that is part of that filtered set.

There is a bit of an issue here though in that the dynamically-populated local field` is called `ChildTasks` while the matching dynamic field is called `SubTasks`. We should probably replace `ChildTasks` every where that exact case-sensitive string is used with `SubTasks` to match the server (even though it can be populated locally with slightly different logic, it is still the same relationship)

Then add `SubTasks` to the default `include` list in the `Task.js` model -- this is the standard set of subfields to include when loading an individual task. The `Tasks` store declares its own proxy to bypass using the Model's proxy so a more no dynamic fields get embedded when loading lists of tasks in the main view, so we can covert `ChildTasks` to `SubTasks` and then load `SubTasks` in the model include spec without conflicting with how the main store loads the Tasks list and then internally populates sub tasks